reactDashboard/src/App.css

```diff
@@ -86,6 +102,22 @@ button.MuiButtonBase-root {
   font-size: 0.75rem;
 }
 
+.darkmode button.MuiButtonBase-root {
```

Comment:
There are better ways of handling Material UI theming, but fine to do that as a separate PR

Reply:
@donal-s I have been keeping a list of the places I have used CSS to style MUI properties.